packages/phishing/src/addrcheck.spec.ts

```diff
@@ -101,9 +101,16 @@ function checkAll (): Promise<[string, string[]][]> {
       'https://dot4.org/promo/',
       'https://dot4.top/promo/'
     ].map((u) => checkTag(u, 'p', 'class="payment-title"')),
+    ...[
```

Comment:
Doubled-up here a bit with checks at 2 different locations - they are a bit flaky since it is either data or a class only, but well, we work with what we have.